instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/JettyClientTracing.java

```diff
@@ -0,0 +1,76 @@
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2;
+
+import static io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2.JettyClientWrapUtil.wrapResponseListeners;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import java.util.List;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.HttpClientTransport;
+import org.eclipse.jetty.client.HttpRequest;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+/**
+ * JettyClientTracing Designed for use outside of Javaagent. Use the static creation methods to
+ * build a Jetty HttpClient that is wrapped with tracing enabled
+ */
+public class JettyClientTracing extends HttpClient {
```

Comment:
Lets go ahead and separate out `TracingHttpClient` instead of having this extend HttpClient directly, it allows this entrypoint and the implementation to be kept separate, and we can make `TracingHttpClient` package private.